go.mod

```diff
@@ -3,63 +3,74 @@ module github.com/BitCannaGlobal/bcna
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.8
-	github.com/cosmos/ibc-go/v3 v3.2.0
+	github.com/cosmos/cosmos-sdk v0.46.1
+	github.com/cosmos/ibc-go/v5 v5.0.0-rc1
 	github.com/gogo/protobuf v1.3.3
 	github.com/golang/protobuf v1.5.2
-	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/ignite/cli v0.23.0
+	github.com/ignite/cli v0.23.1-0.20220902151340-d0fd60f1d4ad
+	github.com/jaekwon/testify v1.6.1
```

Comment:
Same here. I also see that github.com/stretchr/testify is already included, so probably not necessary with this.

Reply:
thanks for the review!

I think that the ignite cli uses the jaekwon code.  Both imports came in when updating the ignite cli, and I believe came from upstream.